android/fakewallet/src/androidTest/res/values/constants.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
this file can be removed, it has no effect (was trying to override the timeout value during testing for faster tests_